src/Api/AdminConsole/Controllers/OrganizationUsersController.cs

```diff
@@ -328,6 +326,23 @@ public async Task Accept(Guid orgId, Guid organizationUserId, [FromBody] Organiz
         }
     }
 
+    private async Task<bool> ShouldHandleResetPasswordAsync(Guid orgId)
+    {
+        var organizationAbility = await _applicationCacheService.GetOrganizationAbilityAsync(orgId);
```

Comment:
@eliykat This change addresses this [comment](https://github.com/bitwarden/server/pull/5319/files#r1928192947).